pkg/efi/manager.go

```diff
@@ -0,0 +1,198 @@
+// nolint:goheader
+
+// This file is part of nullboot
+// Copyright 2021 Canonical Ltd.
+// SPDX-License-Identifier: GPL-3.0-only
```

Comment:
I think we may have an issue here! According to this Apache [document](https://www.apache.org/licenses/GPL-compatibility.html) GPL3 code cannot be included in Apache2.0 one:
```
However, GPLv3 software cannot be included in Apache projects.
The licenses are incompatible in one direction only, and it is a result of ASF's licensing
philosophy and the GPLv3 authors' interpretation of copyright law.
```